segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceBuilder.java

```diff
@@ -66,13 +66,15 @@
 public class ServiceBuilder implements AutoCloseable {
     //region Members
 
-    private static final Duration SHUTDOWN_TIMEOUT = Duration.ofSeconds(30);
+    private static final Duration SHUTDOWN_TIMEOUT = Duration.ofSeconds(45);
     private final SegmentStoreMetrics.ThreadPool threadPoolMetrics;
     private final SegmentToContainerMapper segmentToContainerMapper;
     private final ServiceBuilderConfig serviceBuilderConfig;
     @Getter(AccessLevel.PROTECTED)
     private final ScheduledExecutorService coreExecutor;
     private final ScheduledExecutorService storageExecutor;
+    @Getter(AccessLevel.PUBLIC)
+    private final ScheduledExecutorService lowPriorityExecutor;
```

Comment:
This doesn't appear to be used anywhere.

Reply:
We were using it in `ServiceStarter`. Now, we also use it elsewhere as you can see in the latest push.